google/cloud/storage/bucket_metadata.cc

```diff
@@ -28,6 +28,16 @@ namespace google {
 namespace cloud {
 namespace storage {
 inline namespace STORAGE_CLIENT_NS {
+namespace internal {
+absl::optional<std::string> const& NormalizePap(
+    absl::optional<std::string> const& pap) {
+  static auto const* normalized =
+      new absl::optional<std::string>(PublicAccessPreventionInherited());
```

Comment:
Consider just `new auto(PublicAccessPreventionInherited())`, because I'm not seeing any need to wrap the returned string in an `absl::optional`, just to be unconditionally dereferenced a few lines below.

Reply:
The dereference is (I think) changing from `absl::optional<..> const*` to `absl::optional<..> const&`, right?

Reply:
Oh, I'm sorry. I missed that this was returning a const-ref. My mistake.